app/models/corporate_information_page.rb

```diff
@@ -51,8 +51,16 @@ def search_index
   end
 
   def self.search_only
-    # Ensure only CIPs associated with a live Organisation are indexed in search.
-    super.with_organisation_govuk_status("live")
+    live_govuk_status = super.with_organisation_govuk_status("live")
+
+    accessible_other_govuk_status = super.accessible_documents_policy
+      .with_organisation_govuk_status(%w[joining exempt transitioning])
+
+    accessible_devolved_govuk_status = super.accessible_documents_policy
+      .with_organisation_govuk_status("closed")
+      .where(organisations: { govuk_closed_status: "devolved" })
+
+    live_govuk_status.or(accessible_other_govuk_status).or(accessible_devolved_govuk_status)
```

Comment:
This could be broken onto multiple lines, I think? Would read better:

```suggestion
    live_govuk_status
      .or(accessible_other_govuk_status)
      .or(accessible_devolved_govuk_status)
```

Reply:
Yep, that's a good idea!